### ISSUE TITLE
Cop suggestion: Always call save! / update! instead of save / update

### ISSUE THREAD

**Issue by ndbroadbent** [rubocop-hq/rubocop-rspec#911]
In a Rails app with RSpec tests, `save` can fail silently if there are any validation errors, and it will just return false. The return value is rarely checked in tests. `save!` will crash with an error if anything goes wrong, which is what you would want to happen in your test cases. Otherwise something might not be saving properly and the test is still passing silently, while it should actually be failing.

**Comment by pirj** [comment]
Isn't https://github.com/rubocop-hq/rubocop-rails/blob/master/lib/rubocop/cop/rails/save_bang.rb#L101 doing its job under /spec?

**Comment by ndbroadbent** [comment]
Oh wow, thanks! Yes I just added this to my `.rubocop.yml`:

```
Rails/SaveBang:
  Enabled: true
```

And now RuboCop also automatically converts `save` to `save!` during format-on-save. Whoops, I don't know why `Rails/SaveBang` was missing from my configuration. I do have this:

```
require:
  - rubocop-rspec
  - rubocop-rails
```

Although I think I remember reading something about the default behavior changing so that I have to explicitly opt-in?

Oooh I just found [this documentation](https://github.com/rubocop-hq/rubocop/blob/master/manual/configuration.md#updating-the-configuration-file) and [mry](https://github.com/pocke/mry). Sorry I hadn't seen these before, and will try them out!

Thanks for your time!
